packages/trader/src/Stores/Modules/Trading/Helpers/process.js

```diff
@@ -6,14 +6,16 @@ import * as StartDate from '../Actions/start-date';
 export const processTradeParams = async (store, new_state) => {
     const snapshot = store.getSnapshot();
     const functions = getMethodsList(store, new_state);
+    const promises = [];
 
     // To make sure that every function is invoked and affects the snapshot respectively, we have to use for instead of forEach
     for (let i = 0; i < functions.length; i++) {
         // Shallow copy with Object.assign is good enough to extend the snapshot with new state
         // we don't need deep extension here, since each function in functions array composes a property of the store completely
-        Object.assign(snapshot, await functions[i](snapshot)); // eslint-disable-line no-await-in-loop
+        promises.push(functions[i](snapshot));
     }
 
+    Object.assign(snapshot, await Promise.all(promises));
```

Comment:
<img width="273" alt="image" src="https://user-images.githubusercontent.com/56330681/152908897-316b2667-906d-4c6d-ae7a-526f63d090be.png">
@hirad-rewok Seems to have issues with your test link. It shows ticks in `Multiplier` contract.